newrelic/provider_test.go

```diff
@@ -117,16 +117,16 @@ func TestAccNewRelicProvider_Region(t *testing.T) {
 }
 
 func testAccPreCheck(t *testing.T) {
-	if v := os.Getenv("NEWRELIC_API_KEY"); v == "" {
-		t.Fatal("NEWRELIC_API_KEY must be set for acceptance tests")
+	if v := os.Getenv("NEW_RELIC_API_KEY"); v == "" {
+		t.Log("[WARN] NEW_RELIC_API_KEY has not been set for acceptance tests")
```

Comment:
Why not `t.Fatal` here?

Reply:
Good call, just an oversight when moving things around.